src/software/thunderscope/binary_context_managers/binary_context_managers.py

```diff
@@ -0,0 +1,32 @@
+import psutil
+import google.protobuf.internal.encoder as encoder
+import google.protobuf.internal.decoder as decoder
+
+from software.python_bindings import *
+from proto.import_all_protos import *
+from software.py_constants import *
+from extlibs.er_force_sim.src.protobuf.world_pb2 import (
+    SimulatorState,
+    SimBall,
+    SimRobot,
+)
```

Comment:
Most of these imports can be removed. Same with the other files

Reply:
I've removed most unnecessary imports in accordance with my lsp.